book-store/.idea/book-store.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am following the same pattern which is there in the example which you shared. Please look into the files - App.js and index.js